lib/main_ltp.pm

```diff
@@ -42,9 +42,17 @@ sub load_kernel_tests {
             loadtest_kernel 'update_kernel';
         }
         loadtest_kernel 'install_ltp';
+
+        if (get_var('LIBC_LIVEPATCH')) {
+            die 'LTP_COMMAND_FILE and LIBC_LIVEPATCH are mutually exclusive'
+              if get_var('LTP_COMMAND_FILE');
+            loadtest_kernel 'ulp_openposix';
+        }
+
         # If there is a command file then install_ltp schedules boot_ltp which
         # will schedule shutdown
-        shutdown_ltp() unless get_var('LTP_COMMAND_FILE');
+        shutdown_ltp()
```

Comment:
Maybe update comment above as well?

Reply:
Updated the comment and added the variables to variables.md.